iphone/Maps/Classes/CustomViews/MapViewControls/Search/Console/MWMConsole.mm

```diff
@@ -15,6 +15,9 @@ + (BOOL)performCommand:(NSString *)cmd
 
   if ([self performSound:cmd])
     return YES;
+  
+  if ([self perform3dMode:cmd])
```

Comment:
@VladiMihaylenko @igrechuhin Why do we have separate hidden commands activation code in iOS and in C++ core? Why we can't do it in the C++ core only?